webpack.config.js

```diff
@@ -10,6 +10,8 @@ module.exports = (env, argv) => {
     let releaseMode = argv.mode === 'production';
 
     const config = {
+        node: { fs: 'empty' },
```

Comment:
What does this do?

Reply:
I am not sure why this works. I get this exception when I do not have it:

```
ERROR in ../node_modules/genomic-reader/dist/loader/FileDataLoader.js
Module not found: Error: Can't resolve 'fs' in '/Users/phil/Documents/code/valis/valis-hpgv/node_modules/genomic-reader/dist/loader'
 @ ../node_modules/genomic-reader/dist/loader/FileDataLoader.js 13:13-26
 @ ../node_modules/genomic-reader/dist/loader/index.js
 @ ../node_modules/genomic-reader/dist/index.js
 @ ./track/signal/SignalTileLoader.ts
 @ ./GenomeVisualizer.tsx
 @ ./index.ts

```
I see this reference as a possible known bug this may be related to:
https://github.com/webpack-contrib/css-loader/issues/447

This is a stackoverflow reference to the workaround like the bug reference: 
https://stackoverflow.com/questions/48358748/module-not-found-error-cannot-resolve-module-fs

Reply:
Aha, good to know!